include/seqan3/alphabet/quality/phred63.hpp

```diff
@@ -21,7 +21,7 @@
 namespace seqan3
 {
 
-/*!\brief Quality type for traditional Sanger and modern Illumina Phred scores (full range).
+/*!\brief Quality type for traditional Sanger and modern Illumina Phred scores (full range). <-- ist 93 nicht full range?
```

Comment:
```suggestion
/*!\brief Quality type for traditional Sanger and modern Illumina Phred scores.
```
I would just remove it for now and you can decide in the "Improve Docu PR"  if you add something here again that is more helpful than "full range" or "middle range"